src/main/java/by/matsukiryna/arraytask/service/impl/ArraySortServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package by.matsukiryna.arraytask.service.impl;
+
+import by.matsukiryna.arraytask.entity.ArrayEntity;
+import by.matsukiryna.arraytask.exception.ArrayException;
+import by.matsukiryna.arraytask.service.ArraySortService;
+
+public class ArraySortServiceImpl implements ArraySortService {
+    @Override
+    public void bubbleSort(ArrayEntity arrayEntity) throws ArrayException {
```

Comment:
make a more descriptive name (sorting direction is unclear)

Reply:
I disagree. My mentor asks us to name methods similar to standard libraries